_docs/principles/accessibility.md

```diff
@@ -198,61 +299,67 @@ Helix minimum spacing standards require: 
+Do not use tables for formatting content or for anything other than presenting data. Because a screen reader reads every row and column of a table, including secondary text or data in additional nested rows can make interpreting the contents of a table difficult for the user.
 
 {% endcolumn %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+</div>
 
-#### Failed text inputs
+</section>
 
-There are many acceptable ways to make text inputs visually accessible. If you're indicating a failed text input, make sure to include multiple visual indicators. For example, in addition to the error message, the border color turns red to draw the eye and doubles in thickness from 1px to 2px to provide another distinguishable visual cue that is not reliant on color.
+<section class="static-section"  markdown="1">
 
-It is important to reserve the prominent border at 2px for failed text input as an additional visual indicator for error an state.
+<div class="hxRow"  markdown="1">
 
-{% endcolumn %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+## Interactivity guidelines
 
-{% figure [caption:"Default state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-1.svg"/>
-{% endfigure %}
+Consider the following page layout guidelines when you construct an accessible UI.
 
-{% endcolumn %}
+### Ensure that pointer targets are large enough
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+Pointer targets that are activated by a mouse, trackball, or stylus should be at least **Do we have a size?**. Large pointer targets can be helpful to users who have difficulty with fine motor skills.
 
-{% figure [caption:"Error state"] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-inputs-2.svg"/>
-{% endfigure %}
+### Don't make users hover to find hidden controls
+
+Users that use screen readers or the keyboard to navigate to and select components rely on those components to be visible. Don't hide components as a way to reduce visual clutter. If a control cannot be seen, it cannot be verbally selected.
+
+### Write meaningful alternative text for images
+
+Screen readers read the text you enter in the `<alt>` tag that you associate with an image. If an image provides important contextual information, then use the `<alt>` tag to describe what is happening in the picture. If the image is decorative and does not provide important information, then add an empty `<alt>` tag. If you do not add an `<alt>` tag, some screen readers read the name of the image file, which results in a terrible user experience.
```

Comment:
s/results in a terrible user experience/ which introduces unnecessary confusion and results in a poor user experience.

Reply:
Done.